cleverhans/attacks.py

```diff
@@ -158,7 +164,8 @@ def generate_np(self, x_val, **kwargs):
       raise ValueError("Cannot use `generate_np` when no `sess` was"
                        " provided")
 
-    fixed, feedable, hash_key = self.construct_variables(kwargs)
+    packed = self.construct_variables(kwargs)
+    fixed, feedable, _feedable_types, hash_key = packed
```

Comment:
Looks like `_feedable_types` is unused.
I though it's more common to just use _ for unused values (i.e. `fixed, feedable, _, hash_key = packed`), but I might be wrong.

Reply:
Done